README.md

```diff
@@ -30,64 +30,78 @@ We suggest a couple of alternative ways of installation.
 
 git-machete works under both Python 2.7 and Python 3.x.
 
-### Using Homebrew (macOS)
+<details><summary><b>Using Homebrew (macOS)</b></summary><br/>
```

Comment:
I do not like spoilers. That may be only my personal experience but for me, both look and feel are bad. I am used to non-interactive readmes. `<details> `tags may be good for logs and other content that we might want hide but not for readmes.

Have you ever seen a github readme with spoilers? :thinking: